### DISCUSSION TITLE
推荐一个基于 xray-core 的可运行于 Windows 与 Mac 的客户端程序

### DISCUSSION THREAD

**Discussion by lolo1208** [XTLS/Xray-core#1078] [Show and tell]
[本项目](https://github.com/lolo1208/xray-client) 是基于 [xray-core](https://github.com/XTLS/Xray-core) 的可运行于 Windows 与 Mac（包括 M1 Chip）的客户端程序。可以方便的编辑代理规则，以及查看上下行流量和日志，还可以导入与导出客户端配置。
项目地址：https://github.com/lolo1208/xray-client
![general-1](https://user-images.githubusercontent.com/10737818/169953098-d14814ee-aae7-460e-8c32-5c014e90a0a8.png)
![rules](https://user-images.githubusercontent.com/10737818/169953135-e13a6fca-b237-4b4d-bb52-d3591bc09d90.png)
![logs](https://user-images.githubusercontent.com/10737818/169953149-c9a3c7cf-b08f-4909-a7eb-518d826aaaa8.png)

**Comment by qingniao99** [comment]
你是搞前端的？ 我看是electron

**Reply by lolo1208** [reply]
是的，啥都写写

**Comment by dfldylan** [comment]
怎么没了
